aws-cloudwatch-log-retention-manager/main.tf

```diff
@@ -0,0 +1,117 @@
+locals {
+  tags = {
+    managedBy = "terraform"
+    env       = var.env
+    project   = var.project
+    service   = var.service
+    owner     = var.owner
+  }
+
+  lambda_name = "${var.project}-${var.env}-${var.service}-cloudwatch-retention"
+}
+
+data archive_file lambda {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "archive" in "required_providers"